### PR TITLE
Force a local email to be set for Github repos

### PR DESCRIPTION
**Pull request by andreferreirav2**

### CODE REVIEW

**Comment by kintoandar** [comment]
Updated TOC:

```
Table of Contents
=================

  * [Table of Contents](#table-of-contents)
  * [Collection of git hooks](#collection-of-git-hooks)
    * [pre\-commit hooks](#pre-commit-hooks)
      * [github\_email\_verify](#github_email_verify)
        * [Motivation](#motivation)
        * [Install](#install)
        * [Optional](#optional)
        * [Demo](#demo)
    * [Contributors](#contributors)
```

File: pre-commit/github_email_verify.sh [hunk 1]
```diff
@@ -1,41 +1,49 @@
...
+    git config --local user.email "$GLOBAL_EMAIL"
+    printf "\n[OK] Please run the commit again\n"
+    exit 1
   else
-    read -p "Enter new local user.email: " -r NEW_EMAIL
-
-    read -p "Proceed with new local user.email [$NEW_EMAIL] (y/n): " -r OPTION
+    
+    if [[ ! -z "$PRE_COMMIT_GITHUB_EMAIL" ]]; then
+      read -p "Enter new local user.email (leave empty to use '$PRE_COMMIT_GITHUB_EMAIL'): " -r NEW_EMAIL
```
Comment (kintoandar) [inline pre-commit/github_email_verify.sh]: Sounds a bit better like this:

`read -p "Enter new local user.email or leave blank to proceed with [$PRE_COMMIT_GITHUB_EMAIL]: " -r NEW_EMAIL`

File: README.md [hunk 2]
```diff
@@ -27,6 +27,13 @@ mkdir -p ~/.git-templates/hooks
 git config --global init.templatedir '~/.git-templates'
 ln -s $(pwd)/pre-commit/github_email_verify.sh ~/.git-templates/hooks/pre-commit
 ```
+
+#### Optional
+To add your github's account email as the default email to use, add the following line to your \*rc file (~/.bashrc, ~/.zshrc, etc)
+```
+export PRE_COMMIT_GITHUB_EMAIL=<name@email.domain>
```
Comment (kintoandar) [inline README.md]: Just a little cosmetic change:

`export PRE_COMMIT_GITHUB_EMAIL='user@email.domain'`

File: README.md [hunk 3]
```diff
@@ -27,6 +27,13 @@ mkdir -p ~/.git-templates/hooks
 git config --global init.templatedir '~/.git-templates'
 ln -s $(pwd)/pre-commit/github_email_verify.sh ~/.git-templates/hooks/pre-commit
 ```
+
+#### Optional
+To add your github's account email as the default email to use, add the following line to your \*rc file (~/.bashrc, ~/.zshrc, etc)
```
Comment (kintoandar) [inline README.md]: `In order to have your github's email account as the default email to be used, add the following line to your shell startup file (~/.bashrc, ~/.zshrc, etc.).`

**Comment by kintoandar** [comment]
Thanks so much for your contribution!

**Comment by ruizink** [comment]
Shiny! Don't forget to update the demo.

**Comment by kintoandar** [comment]
On it :D

**Comment by kintoandar** [comment]
We've the new demo live!

**Comment by andreferreirav2** [comment]
Hurray!!

**Pull request merged**